tensorflow/core/util/dump_graph.cc

```diff
@@ -18,6 +18,8 @@ limitations under the License.
 
 #include "tensorflow/core/util/dump_graph.h"
 
+#include <memory>
+#include <unordered_map>
 #include "absl/strings/match.h"
 #include "absl/strings/str_cat.h"
```

Comment:
Can you also include "tensorflow/core/framework/cost_graph.pb.h"? Looks like it's getting an incomplete definition otherwise.

Reply:
Done, add it in dump_graph.h